requests/src/main/scala/weco/api/requests/Main.scala

```diff
@@ -4,7 +4,7 @@ import akka.actor.ActorSystem
 import com.typesafe.config.Config
 import weco.Tracing
 import weco.api.requests.services.{
-  ElasticItemLookup,
```

Comment:
This change ditches the memory version of ItemLookup - implementing a version that worked with the `bySourceIdentifier` function when querying and sorting on works became unwieldy.